src/openms/source/CHEMISTRY/EnzymesDB.cpp

```diff
@@ -53,12 +53,11 @@ namespace OpenMS
   EnzymesDB::EnzymesDB()
   {
     readEnzymesFromFile_("CHEMISTRY/Enzymes.xml");
```

Comment:
within 
```readEnzymesFromFile_```
there is a File::find()
so, the string is correct and it works as long as OpenMS/share is present (OpenMS would not run without it anyways).